configs/mainnet.yaml

```diff
@@ -156,6 +156,8 @@ DOMAIN_SHARD_PROPOSAL: 0x80000000
 DOMAIN_SHARD_COMMITTEE: 0x81000000
 DOMAIN_LIGHT_CLIENT: 0x82000000
 DOMAIN_CUSTODY_BIT_SLASHING: 0x83000000
+# Constant
+NO_SIGNATURE: 0x000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000
```

Comment:
We don't generally put constants that never change into configs as if they are meant to change. Just keeping it in the beacon spec is good enough. Clients will optimize this one way or another anyway, since it never changes.